ddtrace.gemspec

```
@@ -75,4 +75,15 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'datadog-ci', '~> 0.8.1'
 
   spec.extensions = ['ext/datadog_profiling_native_extension/extconf.rb', 'ext/datadog_profiling_loader/extconf.rb']
+
+  spec.post_install_message = <<-MSG
+    Thank you for installing ddtrace.
+
+    As of version 2, the `ddtrace` gem has been renamed to `datadog` to better represent the full suite of Datadog products.
+    The 1.x series will now only receive maintenance updates for security and critical bug fixes.
+
+    To upgrade, please update your Gemfile to replace `ddtrace` with `datadog`.
```

```suggestion
    To upgrade, please use move to the gem `datadog`, instead of `ddtrace`.
```

Do we want to keep the language bundler neutral? In case people are not actually using a Gemfile.